.github/workflows/ci-cd.yml

```diff
@@ -131,9 +139,40 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
           target: runtime-base
-          push: true
-          registry_user: ${{ secrets.ECOSYSTEM_SERVICE_USER_DOCKER_HUB }}
-          registry_token: ${{ secrets.ECOSYSTEM_ACCESS_TOKEN_DOCKER_HUB }}
+          push: false
+          load: true
+      - name: Get EAP from base image
+        id: get_eap_file_name
+        run: |
+          docker cp $(docker create "${{ steps.meta.outputs.full_name }}"):/opt/app .build
+          export EAP_FILE=$(find .build -type f -name "*.eap" -printf "%f\n")
+          delimiter="$(openssl rand -hex 8)"
+          echo "EAP_FILE<<${delimiter}" >> ${GITHUB_OUTPUT}
+          echo "${EAP_FILE}" >> ${GITHUB_OUTPUT}
+          echo "${delimiter}" >> ${GITHUB_OUTPUT}
+      - name: Sign eap-file
+        run: |
+          cd .build
+          curl -XPOST -H 'accept: */*' -H 'Content-Type: multipart/form-data' -H 'Authorization: Bearer ${{secrets.ACAP_PORTAL_SIGNING_BEARER_TOKEN}}' \
+          'https://gw.ext.csi-api.axis.com/ext/acap/admin/application/${{secrets.ACAP_PORTAL_SIGNING_ID}}/sign/binary' -F uploadedFile=@"${{ steps.get_eap_file_name.outputs.EAP_FILE }}" --output Signed_${{ steps.get_eap_file_name.outputs.EAP_FILE }}
```

Comment:
Let's check if we can use repository variables for things like the portal url `https://gw.ext.csi-api.axis.com/ext/acap/admin/application/`

Reply:
I've added a repository variable for the url so please replace `https://gw.ext.csi-api.axis.com/ext/acap/admin/application` with `${{ vars.ACAP_PORTAL_URL }}`